src/components/aiAgent/ChatMessages.tsx

```diff
@@ -0,0 +1,114 @@
+import { Button, Icon, Typography } from 'lago-design-system'
+import { Duration } from 'luxon'
+import { ReactNode } from 'react'
+
+/**
+ * Converts a number (in seconds) into a human-readable duration string
+ * @param seconds - The duration in seconds
+ * @returns A human-readable duration string (e.g., "2m 30s", "1h 15m", "45s")
+ */
+export const formatDuration = (seconds: number): string => {
+  const duration = Duration.fromObject({ seconds }).reconfigure({ locale: 'en-US' })
+  const shifted = duration.shiftTo('minutes', 'seconds').toObject()
```

Comment:
Just curious, what's the benefit of using `.shiftTo('minutes', 'seconds').toObject()` vs `.normalize().toObject()` ?

Also as it's an object returned you could instantiate `minutes` and `secondsStr` directly from this line